docs/reference/commandline/service_update.md

```diff
@@ -171,9 +171,9 @@ $ docker service update --mount-rm /somewhere myservice
 myservice
 ```
 
-### Rolling back to the previous version of a service 
+### Rolling back to the previous version of a service
```

Comment:
@redpanda I realise this wasn't added in your pull request, but can you address this comment as well?